### ISSUE TITLE
[Bug][a11y] There is no scroll functionality on the homescreen and settings and some elements cannot be selected with Talkback enabled

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#3572]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Launch Fenix.
1. Access the home screen or settings.
1. Select the last element from the bottom of the homescreen or settings.
1. Swipe on screen to the right for changing the Talkback position until reaching the top of home screen.

### Expected behavior
The home screen is scrolled up and the top of the homescreen/settings can be accessed.

### Actual behavior
The Talkback doesn't move to the top of the homescreen/settings(the scroll functionality is not working).

### Device information
* Android device: **`Samsung Galaxy Note 9(Android 8.1.0).`**
* Fenix version: **`6/19 1.0.0-rc.2`** 
    1. Notes
* [Video](https://drive.google.com/file/d/1gRRy5S1MtNJRGh0W2Ivnep8zi0m0kID3/view?usp=sharing)
Please note that at the end of the video I scrolled manually in order to reach the top and show that those elements can be selected.

**Comment by mcarare** [comment]
Can QA please retest this and close or update bug description if necessary? TY!

**Comment by sflorean** [comment]
This is still reproducible on Nightly 6/12 following the same steps as in the description, with a small change: should be created collections in order for the screen to be scrollable, given the fact that tabs tray is enabled. 
Device: Samsung Galaxy Note10 (Android 10).
